requirements.txt

```
@@ -25,6 +25,7 @@ loguru==0.6.0
 multidimensional_urlencode==0.0.4
 okta==2.7.0
 openpyxl==3.0.9
+networkx==3.1
```

per comment below, i think even something as simple as this could help?

```suggestion
networkx==3.1 # added to help with privacy preference data migration
```

i'm a bit reluctant to add something like "not used in application runtime" just because at some point we _may_ end up using it in the app, and this then creates an extra maintenance step to remove that note... 🤷 probably thinking too much about this, but i think the takeaway is any small note here could help save some potential confusion and uncertainty down the line!